draft-api/src/main/scala/no/ndla/draftapi/controller/DraftController.scala

```diff
@@ -804,5 +805,39 @@ trait DraftController {
       }
     }
 
+    get(
+      "/frontpage/:slug",
```

Comment:
Samme som for article-api. Ser ingen grunn til å spesifisere at dette er en frontpage-artikkel. Eller er det problematisk å ha denne parallelt med /:id?
```suggestion
      "/:slug",
```

Reply:
Aha. Då gir det meining å ha forskjellig navn. Men bruk heller /article/:slug eller nokke sånt.

Reply:
pathen ligger allerede på /articles/ derfor satt jeg `frontpage` for å skille litt. Hele pathen ser slik ut nå
```
/article-api/v2/articles/frontpage/:slug
/draft-api/v1/drafts/frontpage/:slug
```
Passer ikke helt å ha enda en `articles` i pathen

Reply:
Bruk /slug/:slug då. Dette _kan_ seinere brukes for andre typer enn frontpage-article så vil ikkje ha det hardkoda i pathen.

Reply:
Kva med å endre /:article_id til å håndtere begge? Dersom article_id er en streng så tolkes den som slug mens om det er et tall så brukes det?